tests/components/UniversalResults.stories.tsx

```diff
@@ -26,15 +27,15 @@ const verticalConfigMap = {
   }
 };
 
-export const Primary = (args: UniversalResultsProps) => {
+export const Primary = (args: UniversalResultsProps<DefaultRawDataType>) => {
```

Comment:
Do we need to specify the default? I thought it would default to the default

Reply:
Only StandardSectionProps, StandardCardProps, SectionComponent, and CardComponent have defaults because those are the ones we would expect users may want to write/extend. Vertical/UniversalResults don't because they would generally be able to infer the type from the props. Here we have to specify the type param because we're using the prop interface directly as a type